src/mappings/vaultMappings.ts

```diff
@@ -567,6 +571,93 @@ export function handleTransfer(event: TransferEvent): void {
   }
 }
 
+export function handleUpdateGuardian(event: UpdateGuardian): void {
+  let vaultAddress = event.address;
+  let guardianAddress = event.params.guardian;
+  let vault = Vault.load(event.address.toHexString());
+  if (vault !== null) {
+    log.info('Vault guardian updated. Address: {}, To: {}', [
+      vaultAddress.toHexString(),
+      guardianAddress.toString(),
+    ]);
+    vault.guardian = guardianAddress;
+    vault.save();
+  } else {
+    log.warning('Failed to update guardian of vault {} to {}', [
+      vaultAddress.toHexString(),
+      guardianAddress.toString(),
+    ]);
+  }
+}
+
+export function handleUpdateManagement(event: UpdateManagement): void {
+  let vaultAddress = event.address;
+  let managementAddress = event.params.management;
+  let vault = Vault.load(event.address.toHexString());
+  if (vault !== null) {
+    log.info('Vault management updated. Address: {}, To: {}', [
+      vaultAddress.toHexString(),
+      managementAddress.toString(),
+    ]);
+
+    vault.management = managementAddress;
+    vault.save();
+  } else {
+    log.warning('Failed to update management of vault {} to {}', [
+      vaultAddress.toHexString(),
+      managementAddress.toString(),
+    ]);
+  }
+}
+
+export function handleUpdateGovernance(event: UpdateGovernance): void {
+  let vaultAddress = event.address;
+  let governanceAddress = event.params.governance;
+  let vault = Vault.load(event.address.toHexString());
+  if (vault !== null) {
+    log.info('Vault governance updated. Address: {}, To: {}', [
+      vaultAddress.toHexString(),
+      governanceAddress.toString(),
+    ]);
+
+    vault.governance = governanceAddress;
+    vault.save();
+  } else {
+    log.warning('Failed to update governance of vault {} to {}', [
+      vaultAddress.toHexString(),
+      governanceAddress.toString(),
+    ]);
+  }
+}
+
+export function handleUpdateDepositLimit(event: UpdateDepositLimit): void {
```

Comment:
we could also update vault.tokenDepositLimit here, which represents the total deposit limit; https://github.com/yearn/yearn-vaults-v2-subgraph/issues/161

Reply:
Hmm I don't think I fully understand. there is no `tokensDepositLimit` on the vault and there is also no `totalDepositLimit` on the vault either.

Reply:
Realized I misread a part of your PR; there's a field on the vault entity called `tokensDepositLimit`; your PR implements both depositLimit and availableDepositLimit, making `tokensDepositLimit` redundant. My bad; either you can remove `tokensDepositLimit` or we can do it in a future PR

Reply:
I removed it